recipes/soxs/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "soxs" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/soxs-{{ version }}.tar.gz
+  sha256: 5e585ed8ba4d90572660da8db48dd836dc4def415b30f31efd9e6ebb7c00d43e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy
+    - pip
+    - cython >=0.29.21
+  run:
+    - python
+    - astropy >=3.0
+    - tqdm
+    - pooch
+    - setuptools-scm
```

Comment:
This should normally be not needed at runtime.

Reply:
Hi @xhochy this is fixed now.